coalib/output/ConsoleInteraction.py

```diff
@@ -407,6 +407,16 @@ def print_results_formatted(log_printer,
                 exception)
 
 
+def print_bears_formatted(bears, format=None):
+    format_str = format or ('name:{name}:can_detect:{can_detect}:'
+                            'can_fix:{can_fix}:description:{description}')
+    print('\n\n'.join(format_str.format(name=bear.name,
+                                        can_detect=bear.CAN_DETECT,
+                                        can_fix=bear.CAN_FIX,
+                                        description=bear.get_metadata().desc)
+                      for bear in bears))
```

Comment:
is it possible to have 2 commits, one for implementing `filter-by` for `format` and one for `json`?